judge/pdf_problems.py

```diff
@@ -13,12 +13,15 @@
 
 HAS_SELENIUM = False
 if settings.USE_SELENIUM:
-    from selenium import webdriver
-    from selenium.common.exceptions import TimeoutException
-    from selenium.webdriver.common.by import By
-    from selenium.webdriver.support import expected_conditions as EC
-    from selenium.webdriver.support.ui import WebDriverWait
-    HAS_SELENIUM = True
+    try:
+        from selenium import webdriver
+        from selenium.common.exceptions import TimeoutException
+        from selenium.webdriver.common.by import By
+        from selenium.webdriver.support import expected_conditions as EC
+        from selenium.webdriver.support.ui import WebDriverWait
+        HAS_SELENIUM = True
+    except ImportError:
+        pass
```

Comment:
```suggestion
        HAS_SELENIUM = False
```

probably also want to print the stacktrace

Reply:
Import failure is now logged. It's also done as a warning (as opposed to error) because this can happen under certain normal circumstances.